static/app/utils/demoMode.tsx

```diff
@@ -1,8 +1,6 @@
-import getCookie from 'sentry/utils/getCookie';
-
 export function extraQueryParameter(): URLSearchParams {
-  // cookies that have = sign are quotes so extra quotes need to be removed
-  const extraQueryString = getCookie('extra_query_string')?.replaceAll('"', '') || '';
+  const extraQueryString =
+    window.SandboxData?.extraQueryString?.replaceAll('"', '') || '';
```

Comment:
@Neo-Zhixing Do we still need to do this `replaceAll` if we aren't ready from a cookie?

Reply:
Not sure why we were doing replaceAll originally

Reply:
@Neo-Zhixing See the comment: 
```
// cookies that have = sign are quotes so extra quotes need to be removed
```
It's a cookie specific issue